ddtrace/contrib/internal/langgraph/patch.py

```diff
@@ -222,10 +303,14 @@ def patch():
 
     wrap(RunnableSeq, "invoke", traced_runnable_seq_invoke(langgraph))
     wrap(RunnableSeq, "ainvoke", traced_runnable_seq_ainvoke(langgraph))
+    wrap(RunnableSeq, "astream", traced_runnable_seq_astream(langgraph))
     wrap(Pregel, "stream", traced_pregel_stream(langgraph))
     wrap(Pregel, "astream", traced_pregel_astream(langgraph))
     wrap(PregelLoop, "tick", patched_pregel_loop_tick(langgraph))
 
+    if get_version() >= "0.3.29":
+        wrap(langgraph.utils.runnable, "_consume_aiter", traced_runnable_seq_consume_aiter(langgraph))
```

Comment:
Same with `_consume_iter()`. Do we need to trace sync cases?

Reply:
Same as the above I think - since this function is only used in the context of `RunnableSeq.stream()`, and that's not used, I don't think we need it here too